server/views/pages/overview/partials/overviewTab/overviewTabContentsV2.njk

```diff
@@ -12,138 +12,154 @@
   {% block content %}
     <div class="govuk-grid-column-two-thirds app-u-print-full-width">
       {% if not isPostInduction and not problemRetrievingData %}
-          <section data-qa="pre-induction-overview">
-            {{ govukNotificationBanner({
-              html: createInductionBanner
-            }) }}
-          </section>
+        <section data-qa="pre-induction-overview">
```

Comment:
This looks like a lot of change, but mostly it was indentation changes.
Once the code was indented in the same way throughout, it was easy to identify a rouge closing div (towards the bottom), and that the action menu and it's 1/3rd div was contained within the wrong div (it was previously within the 2/3rds div)